.github/workflows/duckdb.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       #     path: integration/duckdb/build/lance.duckdb_extension
       #     retention-days: 1
   MacOS:
-    runs-on: macos-13
+    runs-on: macos-13-xlarge
```

Comment:
Thanks @judahrand . We'd like to see how fast this goes, and see whether  it makes economical sense giving the M1 pricing.

Also we might want to still keep the intel runner. To make sure Lance works on both Intel and Apple Silicon mac for the timebeing

Reply:
I guess we dont really need duckdb to be on apple silicon because it is too expensive :)